### DISCUSSION TITLE
Group-level analyses

### DISCUSSION THREAD

**Discussion by a-ferfer** [HALFpipe/HALFpipe#751] [General]
Hi,
I'm looking for documentation on group level analyses, in essence I'm looking to run a Group ICA analyses with Halfpipe. Do you have the information?

Best,
Andrea

**Comment by nadza-dz** [comment]
Hi Andrea, we are currently working on updating our main page to include more information specifically about running group level analyses, and we hope that this will be up very very soon. In the meantime, can you maybe explain a bit more what you mean with Group ICA analyses? Is there something specific you are struggling to implement in HALFpipe? 

Best,
Nadza

**Comment by a-ferfer** [comment]
Hi Nadza, thank you very much for the quick reply. My question was tailored to steps after preprocessing analyses have been done. Does Halfpipe facilitates post-processing analyses for group level ICA, such as dual regression similar to FSL? I'm trying to find documentation on how to run those analyses.
Best,
Andrea
